deployment/solana/initialize.ts

```diff
@@ -15,19 +14,14 @@ async function initializeSolanaTbr(
 ): Promise<void> {
   const signerKey = new PublicKey(await signer.getAddress());
   const connection = getConnection(chain);
-  const solanaDependencies = dependencies.find((d) => d.chainId === chain.chainId);
-  if (solanaDependencies === undefined) {
-    throw new Error(`No dependencies found for chain ${chain.chainId}`);
-  }
-  const tbr = new SolanaTokenBridgeRelayer(
-    { connection },
-    {
-      tokenBridgeProgramId: new PublicKey(solanaDependencies.tokenBridge),
-      wormholeProgramId: new PublicKey(solanaDependencies.wormhole),
-    },
-  );
+  const tbr = new SolanaTokenBridgeRelayer({ connection });
 
-  const initializeIx = await tbr.initialize(signerKey);
+  const tmpError = () => { throw new Error('TODO: add this field'); };
```

Comment:
should this be implemented before merging?

Reply:
I don't really know how you handle the deployment, that's why I didn't really know how to fix it.